channels-rke2.yaml

```
@@ -355,3 +367,15 @@ releases:
       rke2-canal:
         repo: rancher-rke2-charts
         version: v3.20.3-build2022011406
+  - version: v1.22.7-rc2+rke2r1
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-22-7-rc2-rke2r1
+      <<: *serverArgs-v1-22-6-rke2r1
+    agentArgs: &agentArgs-v1-22-7-rc2-rke2r1
+      <<: *agentArgs-v1-22-6-rke2r1
+    charts: &charts-v1-22-7-rc2-rke2r1
+      <<: *charts-v1-22-6-rke2r1
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.20.3-build2022011406
```

Actually it looks like it's changed: https://github.com/rancher/rke2/blob/release-1.22/Dockerfile#L102
```suggestion
        version: v3.21.4-build2022020801
```